src/styles/bar.css

```diff
@@ -63,8 +70,21 @@
   text-indent: -9999em;
 }
 
-@media (min-width: 30.0625em) {
-  [role=banner] .coz-bar-burger {
+@media (min-width: 48.0625em) {
+  [role=banner] .coz-bar-burger,
+  [role=banner] .coz-drawer-wrapper {
+    /* Fix unwanted `!important` in
+     * https://github.com/cozy/cozy-ui/blob/v3/stylus/ui-base/boxes.styl#L9
+     *
+     * FIXME: remove it when hide() method is removed from cozy-ui
+     */
+    display: none !important;
```

Comment:
It's now merged in cozy-ui, with the next release I think that you can update this part.